### PR TITLE
postman-sandbox

### PR DESCRIPTION
**Pull request by urosjarc**
For more informations: https://github.com/BigstickCarpet/postman-bdd/issues/6

### CODE REVIEW

**Comment by urosjarc** [comment]
@urosjarc: Todo: Add sandbox dependencies to types in tsconfig.

File: types/postman-sandbox/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,52 @@
+// Type definitions for Postman sandbox.
+// Project: https://www.getpostman.com/docs/sandbox
+// Definitions by: Uroš Jarc <https://github.com/urosjarc/>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/**
+ * ENV
+ */
+declare const environment: Object;
```
Comment (yuit) [inline types/postman-sandbox/index.d.ts]: Don't use`Object` type here. better to use `object` . You will want to add `// TypeScript Version: 2.2` for the header as `object` is 2.2 feature

File: types/postman-sandbox/index.d.ts [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+declare function setEnvironmentVariable(name: string, value: string): void;
+declare function getEnvironmentVariable(name: string): string;
+declare function clearEnvironmentVariable(name: string): void;
+declare function clearEnvironmentVariables(): void;
+
+/**
+ * GLOBALS
+ */
+declare const globals: Object;
```
Comment (yuit) [inline types/postman-sandbox/index.d.ts]: Don't use`Object` type here. better to use `object`

File: types/postman-sandbox/index.d.ts [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+declare function setGlobalVariable(name: string, value: string): void;
+declare function getGlobalVariable(name: string): string;
+declare function clearGlobalVariable(name: string): void;
+declare function clearGlobalVariables(): void;
+
+/**
+ * REQ
+ */
+declare const request: {
+  data: Object;
```
Comment (yuit) [inline types/postman-sandbox/index.d.ts]: Don't use`Object` type here. better to use `object`
(below as well)

File: types/postman-sandbox/postman-sandbox-tests.ts [hunk 4]
```diff
@@ -0,0 +1,50 @@
+let isString: string;
```
Comment (yuit) [inline types/postman-sandbox/postman-sandbox-tests.ts]: nit: the naming makes me expect the type to be boolean instead

File: types/postman-sandbox/tsconfig.json [hunk 5]
```diff
@@ -0,0 +1,22 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "es5"
+        ],
+        "noImplicitAny": true,
+        "noImplicitThis": true,
+        "strictNullChecks": false,
```
Comment (andy-ms) [inline types/postman-sandbox/tsconfig.json]: Make this `true`.

File: types/postman-sandbox/postman-sandbox-tests.ts (hunk 4 above)
Comment (andy-ms) [inline types/postman-sandbox/postman-sandbox-tests.ts]: It's better to use [`$ExpectType`](https://github.com/Microsoft/dtslint#write-tests) than this style anyway.

**Comment by mhegazy** [comment]
Thanks for your contribution. This PR has failing CI tests and can not be merged in at the time being. For housekeeping purposes we are closing stale PRs. If you'd still like to merge this code in, please open a new PR that has been merged and rebased with the `master` branch.

**Pull request closed without merging**